src/MCPClient/lib/clientScripts/archivematicaCreateMETSMetadataCSV.py

```diff
@@ -43,7 +51,7 @@ def parseMetadata(SIPPath):
     See parseMetadataCSV for details on parsing.
 
     :param SIPPath: Path to the SIP
-    :return: {<filename>: OrderedDict{<metadata name>=<metadata value>} }
+    :return: {<filename>: OrderedDict(key: [values]) }
```

Comment:
Yeah, I'd say either document the actual return type or convert to an `OrderedDict` before returning.